scripts/linting/lint-changed-md-files.sh

```diff
@@ -1,13 +0,0 @@
-#!/usr/bin/env bash
-
-set -euo pipefail
-
-# lint_modified_markdown_files runs the linter only if changes have been made to any md files.
-function lint_modified_markdown_files() {
```

Comment:
was this supposed to be removed?

Reply:
Yes, the [new linter workflow](https://github.com/cosmos/ibc-go/blob/feat/migrate-fully-to-docusaurus/.github/workflows/markdown-lint.yml) only runs if there are changed markdown files and only on changed markdown files.

Also the new markdown linter fast enough ( < 1 second ) so that we don't need to do this locally either